Makefile

```diff
@@ -173,8 +173,7 @@ generators:
 	python -m aea.cli hash all --packages-dir=./tests/data/packages
```

Comment:
tox should be used for all commands here. We already have the equivalent tox commands I think

Reply:
I introduced them